src/mono/wasm/debugger/BrowserDebugProxy/MonoSDBHelper.cs

```diff
@@ -897,7 +897,7 @@ public async Task<bool> Step(SessionId sessionId, int thread_id, StepKind kind,
             commandParamsWriter.Write((byte)1);
             commandParamsWriter.Write((byte)ModifierKind.Step);
             commandParamsWriter.Write(thread_id);
-            commandParamsWriter.Write((int)0);
+            commandParamsWriter.Write((int)1);
```

Comment:
What does 0, and 1 here mean? named consts might be helpful.

Reply:
It's the size of the step, before we were using `minimal` and debugger-libs uses `line`